pkgs/development/python-modules/application/default.nix

```diff
@@ -1,21 +1,26 @@
-{ lib, buildPythonPackage, fetchdarcs, zope_interface, isPy3k }:
+{ lib, buildPythonPackage, fetchFromGitHub, zope_interface, isPy3k }:
 
 buildPythonPackage rec {
   pname = "python-application";
-  version = "2.7.0";
+  version = "2.8.0";
```

Comment:
This is used by exactly 1 other package in all of NixPkgs, and that package actually asserts that we need  2.8.0:
https://github.com/AGProjects/python-otr/blob/master/setup.py#L22